.github/workflows/build.yml

```diff
@@ -156,6 +152,8 @@ jobs:
       fail-fast: false
       matrix:
         java: [ 23, 17 ]  # If we change these, make sure to adjust ci-complete.yml
+    outputs:
+      timed-out: ${{ (steps.junit-test.outputs.gradle-exitcode == '124' || steps.junit-quarantined-test.outputs.gradle-exitcode == '124') }}
```

Comment:
it seems line#221 can use this output?

Reply:
hm, maybe not. I think the job outputs aren't evaluated until the job is complete.